docs/getting-started.md

```diff
@@ -1,37 +1,6 @@
 Getting Started
 ===============
+To get started, follow the guides below to install and/or develop CommCare Sync.
 
-To get up and running on a local environment, follow the steps below.
-
-## Install Ansible
-
-From the [Ansible Installation Guide](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-on-ubuntu),
-run the following on your local/control machine
-
-```bash
-sudo apt-add-repository ppa:ansible/ansible
-sudo apt-get update
-sudo apt-get install ansible
-```
-
-## Development
-
-Development for this tool is set up to run on a Vagrant VM.
-To develop and test, follow the instructions below.
-
-### Set up Vagrant
-
-[Follow this guide](https://www.vagrantup.com/docs/installation/)
-
-### Deploy to local VM
-
-*From your local machine*
-
-```bash
-vagrant up --provision
-```
-
-This should download and deploy the latest commcare-sync code to a local Vagrant VM.
-
-If everything is successful you can load [http://192.168.11.10/](http://192.168.11.10/) in a browser
-and get going!
+* [Set up CommCare Sync](install-commcare-sync.md)
+* [Set up development environment](development.md)
```

Comment:
I like the idea of breaking this out, but I'm a little confused by the execution. If you're doing development it seems like you need to follow some - but not all - of the steps in the "set up" docs, and then follow the "dev environment" docs (see comments below). In production I think you'd *just* follow the "set up" docs, yeah?

I wonder if it might be easier to create two totally different pages for development vs setting up a new enviornment?